h/views/main.py

```diff
@@ -87,7 +87,10 @@ def _html_link(request, annotation):
 
 def _incontext_link(request, annotation):
```

Comment:
I could use suggestions on how best to test link handling. This is registered via the `includeme()` function below, so one approach would perhaps be to call `includeme()` with a mock context object and intercept the link handler registration. Thoughts?